# Generated for NaoQi method ALVisionToolbox.isVideoRecording
# isVideoRecording

---
# True/False
bool value


